packages/dev/core/src/Engines/constants.ts

```diff
@@ -638,9 +638,9 @@ export class Constants {
 
     /**
      * Constant used to retrieve clip-space (non-linear) depth index in the textures array in the prepass
-     * using the getIndex(Constants.PREPASS_NDC_DEPTH_TEXTURE_TYPE)
+     * using the getIndex(Constants.PREPASS_HYPERBOLIC_DEPTH_TEXTURE_TYPE)

```

Comment:
I think PREPASS_SCREENSPACE_DEPTH_TEXTURE_TYPE is better, because the Z coordinate in NDC is already hyperbolic, as the only difference between NDC and screen depth is a -1..1 to 0..1 scaling (at least in WebGL). So, we would not know if "HYPERBOLIC" means NDC or SCREEN space.

Also, you should modify the doc "Constant used to retrieve clip-space (non-linear) depth index in the textures array in the prepass" to something like "Constant used to retrieve screen-space (non-linear values between 0 and 1) depth index in the textures array in the prepass".

And sorry for this new change, because I've already made you change CLIP to NDC!

Reply:
Good suggestion. Done.